tests/test_message.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+# This is a hack for now until directory structure is sorted
+# When this is removed remember to un-ignore E402 in flake8
+import sys
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+PROJECT_DIR = os.path.abspath(os.path.join(TEST_DIR, os.pardir))
+sys.path.insert(0, PROJECT_DIR)
```

Comment:
This adds the directory above this one (where the message.py files are) to the system PATH, so we can use `import message` in the next line. 

We get this directory "TEST_DIR" (which is the directory containing this script, referred to here with __file__).  Then we use that to find the path of the directory where message.py is.  Then we tell the code to add that to the path.

The right way to import message.py would be organizing the repo into a correct package structure so you can do relative imports, but including that change here would make this PR mess.  I also need to read more about the right way to structure it.